public/app/core/components/TraceToProfiles/TraceToProfilesSettings.tsx

```diff
@@ -55,7 +55,9 @@ export function TraceToProfilesSettings({ options, onOptionsChange }: Props) {
       supportedDataSourceTypes.includes(dataSource.type) &&
       dataSource.uid === options.jsonData.tracesToProfiles?.datasourceUid
     ) {
-      dataSource.getProfileTypes().then((profileTypes) => {
+      const end = Date.now();
+      const start = end - 24 * 60 * 60 * 1000; // 1 day ago
+      dataSource.getProfileTypes(start, end).then((profileTypes) => {
```

Comment:
@joey-grafana I am not entirely sure if this is a good thing to do and what its impact is